web-local/src/lib/svelte-query/dashboards.ts

```diff
@@ -9,17 +9,15 @@ export function useDashboardNames(repoId: string) {
   return useRuntimeServiceListFiles(
     repoId,
     {
-      glob: "{sources,models,dashboards}/*.{yaml,sql}",
+      glob: "{dashboards}/*.{yaml}",
```

Comment:
Any reason to revert back to 3 calls every second? This was written like it was to make it so that there is only one call.

I think the best option here would be to have a query that gets all files and maps to name. Individual queries can then filter out what they need.

Reply:
Ahh I didn't pick up on that optimization. I mistakenly thought it was leftover from a time when the runtime hadn't implemented glob support. Then I'm still not understanding why we provide this catchall glob pattern – does it make sense to remove it altogether?

You're right, we should avoid successive calls to the API. I'll edit.

Reply:
We need the glob, otherwise runtime might scan the entire dir :) Will be a disaster when it can have node_modules. Until we have safeguards in runtime lets keep it.